modules/AWS/eks/locals.tf

```diff
@@ -20,5 +20,6 @@ locals {
   eks_node_policies = [ "arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore",
                         "arn:aws:iam::aws:policy/AmazonEKSWorkerNodePolicy",
                         "arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly",
-                        "arn:aws:iam::aws:policy/AmazonEKS_CNI_Policy" ]
+                        "arn:aws:iam::aws:policy/AmazonEKS_CNI_Policy",
+                        "arn:aws:iam::aws:policy/service-role/AmazonEBSCSIDriverPolicy" ]
```

Comment:
This one is required for ebs csi driver to attach EBS vols

Reply:
Was this always missing then, just wondering how it would have ever worked?

Reply:
Previously, ebs attachment was done by the apiserver/controller (which has the powers), while now it's a separately deployed service that runs on the node.